source/taskmanager/src/main/java/nl/aerius/taskmanager/PriorityTaskSchedulerFileHandler.java

```diff
@@ -53,26 +49,16 @@ public PriorityTaskSchedule read(final File file) throws IOException {
   }
 
   private PriorityTaskSchedule readFromFile(final File file) throws IOException {
-    try (final Reader reader = Files.newBufferedReader(file.toPath(), StandardCharsets.UTF_8)) {
-      return gson.fromJson(reader, PriorityTaskSchedule.class);
-    }
+    return objectMapper.readValue(file, PriorityTaskSchedule.class);
   }
 
-  private PriorityTaskSchedule readFromEnvironment(final String workerQueueName) {
+  private PriorityTaskSchedule readFromEnvironment(final String workerQueueName) throws JsonMappingException, JsonProcessingException {
```

Comment:
```suggestion
  private PriorityTaskSchedule readFromEnvironment(final String workerQueueName) throws IOException {
```
Think these are subclasses of `IOException` at least.

Reply:
Good point. Turns out `JsonMappingException` is a subclass of `JsonProcessingException`. Therefore I can reduce this to `JsonProcessingException`. They are indeed subclasses of `IOException`. But with only 1 exception left I keep that one.